packages/api/src/platforms/vtex/utils/cookies.ts

```diff
@@ -0,0 +1,5 @@
+import type { Context } from '../index'
+
+export const setCookie = (headers: Headers, ctx: Context) => {
+  ctx.storage.cookies = headers?.get('set-cookie') ?? ''
```

Comment:
Aren't we erasing cookies if no `set-cookies` header is sent? Is this the expected behavior?

Reply:
Good question @gvc, I'd also ask if is it not the case of use `getSetCookie` function here, to get all the response-related Set-Cookie items?

Reply:
> Aren't we erasing cookies if no set-cookies header is sent? Is this the expected behavior?

On my understanding, it would be set in a "request scope", because I was thinking about how would it be to unset, but the expire date should do it for us. I believe this function needs more thought, I'll circle back to you later. Thanks for the question!

> I'd also ask if is it not the case of use getSetCookie function here, to get all the response-related Set-Cookie items? Because we can have more than one, can't we?

Yes, I was having typescript issues when trying to use [getSetCookie](https://developer.mozilla.org/en-US/docs/Web/API/Headers/getSetCookie), I'll investigate it more!

Reply:
@gvc We don't need to worry about cookies that don't send `set-cookies` in the header, because the cookies are used to communicate between services, we aren't a service, but the "channel". It is not our responsibility to take a cookie and add it to a specific call. We just receive everything we can control (just save it on the browser) and send it back to whoever is interested (service - backend) uses it.

If we start getting cookies without the set-cookie, we create in our layer an abstraction and business rule that the context belongs to another product.

Reply:
I agree with @lucasfp13 in cases where there is more than one cookie, we don't work well and we should support it.